system-int-test/src/cmr/system_int_test/utils/bootstrap_util.clj

```diff
@@ -154,13 +154,41 @@
    (bulk-index-grids nil {transmit-config/token-header (transmit-config/echo-system-token)}))
   ([provider-id]
    (bulk-index-grids provider-id {transmit-config/token-header (transmit-config/echo-system-token)}))
-  ([provider-id headers] 
+  ([provider-id headers]
    (bulk-index-by-url
     (if (nil? provider-id)
       (url/bulk-index-grids-url)
       (url/bulk-index-grids-url provider-id))
     headers)))
 
+(defn bulk-index-order-options
+  "Call the bootstrap app to bulk index order options (either all of them, or just the
+  ones for the given provider)."
+  ([]
+   (bulk-index-order-options nil {transmit-config/token-header (transmit-config/echo-system-token)}))
+  ([provider-id]
+   (bulk-index-order-options provider-id {transmit-config/token-header (transmit-config/echo-system-token)}))
+  ([provider-id headers]
+   (bulk-index-by-url
+    (if (nil? provider-id)
+      (url/bulk-index-order-options-url)
+      (url/bulk-index-order-options-url provider-id))
+    headers)))
+
+(defn bulk-index-generics
+  "Call the bootstrap app to bulk index a generic (either all of them, or just the
+  ones for the given provider)."
+  ([concept-type]
+   (bulk-index-generics concept-type {transmit-config/token-header (transmit-config/echo-system-token)}))
```

Comment:
what does this call? the function below wakes a concept type and a provider id not a token, the last one takes 3 parameters, so what is this doing?

Reply:
It just sends the concept-type to `system-int-test/src/cmr/system_int_test/utils/url_helper.clj` which makes the route that then gets passed to the `bulk-index-by-url` function to actually get executed alongside the headers to initiate the bulk-index. I think that the first case needs to pass a nil for the provider-id. I hadn't noticed that since I only call the function with the provider specified. Correcting now